--- srt-1.5.1/CMakeLists.txt.orig	2023-04-04 22:45:12.405917093 +0200
+++ srt-1.5.1/CMakeLists.txt	2023-04-05 06:25:29.629542138 +0200
@@ -68,15 +68,12 @@ if (NOT MICROSOFT)
 	# Set CMAKE_BUILD_TYPE properly, now that you know
 	# that ENABLE_DEBUG is set as it should.
 	if (ENABLE_DEBUG EQUAL 2)
-		set (CMAKE_BUILD_TYPE "RelWithDebInfo")
 		add_definitions(-DNDEBUG)
 	elseif (ENABLE_DEBUG) # 1, ON, YES, TRUE, Y, or any other non-zero number
-		set (CMAKE_BUILD_TYPE "Debug")
 
 		# Add _DEBUG macro in debug mode only, to enable SRT_ASSERT().
 		add_definitions(-D_DEBUG)
 	else()
-		set (CMAKE_BUILD_TYPE "Release")
 		add_definitions(-DNDEBUG)
 	endif()
 endif()
