--- srt-1.5.4/CMakeLists.txt.orig	2025-03-23 16:19:37.209633897 +0100
+++ srt-1.5.4/CMakeLists.txt	2025-03-23 16:21:38.892308019 +0100
@@ -73,7 +73,6 @@ if (NOT MICROSOFT)
 	# Set CMAKE_BUILD_TYPE properly, now that you know
 	# that ENABLE_DEBUG is set as it should.
 	if (ENABLE_DEBUG EQUAL 2)
-		set (CMAKE_BUILD_TYPE "RelWithDebInfo")
 		if (ENABLE_ASSERT)
 			# Add _DEBUG macro if explicitly requested, to enable SRT_ASSERT().
 			add_definitions(-D_DEBUG)
@@ -81,12 +80,10 @@ if (NOT MICROSOFT)
 			add_definitions(-DNDEBUG)
 		endif()
 	elseif (ENABLE_DEBUG) # 1, ON, YES, TRUE, Y, or any other non-zero number
-		set (CMAKE_BUILD_TYPE "Debug")
 
 		# Add _DEBUG macro in debug mode only, to enable SRT_ASSERT().
 		add_definitions(-D_DEBUG)
 	else()
-		set (CMAKE_BUILD_TYPE "Release")
 		add_definitions(-DNDEBUG)
 	endif()
 endif()
